workflow-service-sdk/pom.xml

```diff
@@ -173,7 +175,12 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-web</artifactId>
         </dependency>
-
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>${lombok.version}</version>
+            <scope>provided</scope>
```

Comment:
isn't this needed for runtime of the SDK?

Reply:
This is what Lombok [website](https://projectlombok.org/setup/maven) suggests to configure the dependency:

> To set up lombok with any build tool, you have to specify that the lombok dependency is required to compile your source code, but does not need to be present when running/testing/jarring/otherwise deploying your code. Generally this is called a 'provided' dependency.

Do you think that in our project the scope should be different?

Reply:
No. Provided makes sense based on the description.